BenchmarkVCFs/CompareBenchmarks.wdl

```diff
@@ -403,7 +409,7 @@ def main(sample_ids, configurations, summaries, stratifiers, order_of_samples, o
     if stratifiers is None:
         stratifiers = data['Stratifier'].unique()
     else:
-        stratifiers = ['all'] + stratifiers
+        stratifiers = ['WholeGenome'] + stratifiers
```

Comment:
If you end up changing the label above, also change it here. There are a few more in FEEvaluation and F1Evaluation